### DISCUSSION TITLE
Rad Pro: Random lottery number generator

### DISCUSSION THREAD

**Discussion by donavi-amoris** [Gissio/radpro#157] [Q&A] [answered]
It would be nice to create a lottery number generator for Eurojackpot or DE lottery.
Or make a universal interface for entering any range of numbers 6 of 49 or 5 of 50 to select any range in the lottery and display it on the display like 3.44.11.13.46

**Comment by Gissio** [comment, marked as answer]
No need for that. To generate a random number from 1 to 49, for example, you can use the following procedure:

1. Generate two decimal RNG digits.
2. If the digits fall within the range 1 to 49, keep it as your random number and stop.
3. Otherwise, go back to step 1.

For example, suppose your RNG gives you these decimals digits: 1274361926759157. The first two digits are 12, so this is your first number. The second two digits are 74, so you discard them and go for the next two, which are 36, so there is your second number. And so on.

Do not reuse digits. It will skew probabilities.
